packages/node_modules/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -706,6 +706,89 @@ skipInNode(describe)('plugin-meetings', () => {
           throw e;
         }));
 
+      it('alice resolution is 1080p, check what bob receives', (done) => {
+        testUtils.delayedPromise(alice.meeting.setLocalVideoQuality('1080p'))
+          .then(() => {
+            testUtils.waitUntil(2000);
+            const remoteSettings = bob.meeting.mediaProperties.remoteVideoTrack.getSettings();
+            const localSettings = alice.meeting.mediaProperties.videoTrack.getSettings();
+
+            assert.equal(localSettings.height, '1080');
+            assert.equal(remoteSettings.height, '1080');
+            done();
+          }).catch((error) => {
+            console.error('Alice resolution 1080p: ', error);
+
+            done();
+          });
+      });
+
+      it('alice resolution is 720p, check what bob receives', (done) => {
+        testUtils.delayedPromise(alice.meeting.setLocalVideoQuality('720p'))
+          .then(() => {
+            testUtils.waitUntil(2000);
+            const remoteSettings = bob.meeting.mediaProperties.remoteVideoTrack.getSettings();
+            const localSettings = alice.meeting.mediaProperties.videoTrack.getSettings();
+
+            assert.equal(localSettings.height, '720');
+            assert.equal(remoteSettings.height, '720');
+            done();
+          }).catch((error) => {
+            console.error('Alice resolution 720p: ', error);
+
+            done();
+          });
+      });
+
+      it('alice downgrades to 480p, check what bob receives', (done) => {
+        testUtils.delayedPromise(alice.meeting.setLocalVideoQuality('480p'))
+          .then(() => {
+            testUtils.waitUntil(2000);
+            const remoteSettings = bob.meeting.mediaProperties.remoteVideoTrack.getSettings();
+
+            assert.equal('480', remoteSettings.height);
+            done();
+          }).catch((error) => {
+            console.error('Alice downgrade to 480p: ', error);
+
+            done();
+          });
+      });
+
+      it('bob changes remote to 480p, check what bob receives', (done) => {
+        testUtils.delayedPromise(alice.meeting.setLocalVideoQuality('720p'))
+          .then(() => {
+            testUtils.waitUntil(2000);
+            testUtils.delayedPromise(bob.meeting.setRemoteQualityLevel('LOW'))
+              .then(() => {
+                testUtils.waitUntil(3000);
```

Comment:
Are we sure this time is sufficient for the action to happen !?

Reply:
It was not, added an interval.